imagersite/imagersite/tests.py

```diff
@@ -49,35 +169,141 @@ def test_register_user(self):
 class ImagesRoutesTests(TestCase):
     """Class for testing views."""
 
-    def test_library(self):
-        """Test library route."""
+    @classmethod
+    def setUpClass(cls):
+        """Test setup."""
+        super(TestCase, cls)
+        # fake = Faker()
+        for _ in range(1):
+            user = UserFactory.create()
+            user.set_password(factory.Faker('password'))
+            user.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        """Tear down for test."""
+        super(TestCase, cls)
+        User.objects.all().delete()
+
+    def test_library_status(self):
+        """Test library route status code."""
+        user = User.objects.first()
+        self.client.force_login(user)
         response = self.client.get('/images/library/')
-        self.assertEqual(response.status_code, 302)
+        self.assertEqual(response.status_code, 200)
+
+    def test_library_template(self):
+        """Test library route template."""
+        user = User.objects.first()
+        self.client.force_login(user)
+        response = self.client.get('/images/library/')
+        self.assertEqual(response.templates[0].name, 'imager_images/library.html')
+
+    def test_library_base_template(self):
+        """Test library route base template."""
+        user = User.objects.first()
+        self.client.force_login(user)
+        response = self.client.get('/images/library/')
+        self.assertEqual(response.templates[1].name, 'generic/base.html')
+
+    def test_profile_status(self):
```

Comment:
These profile tests should also not be here, but in the `imager_profile` app tests.